### PR TITLE
New folder structure + common spark image + correct networking instructions

### PR DESCRIPTION
**Pull request by mdrumond**
- Moving to new folder structure (with docker image tags marking version instead of type of images)

- Changing benchmar to use common spark image, the same used by the in-memory analytics benchmark

- Fixing networking instructions, instructing users to use --net.

Other minor changes to make it easier to use this image in perfkit.

### CODE REVIEW

File: benchmarks/in-memory-analytics/README.md [hunk 1]
```diff
@@ -36,63 +36,66 @@ More information about the dataset is available at
...
-be used to tweak execution. For example, to ensure that Spark has enough memory
-allocated to be able to execute the benchmark in-memory, supply it with
---driver-memory and --executor-memory arguments:
+Any arguments after the two mandatory ones are passed to spark-submit
+and can be used to tweak execution. For example, to ensure that Spark
+has enough memory allocated to be able to execute the benchmark
+in-memory, supply it with --driver-memory and --executor-memory
+arguments:
 
-    $ docker run --rm --volumes-from data cloudsuite/in-memory-analytics \
```
Comment (neo-apz) [inline benchmarks/in-memory-analytics/README.md]: Good job. :+1: 
Why did you change the `in-memory-analytics` README file?
Also, could you please give me these details:
Path to all dockerfiles with their repo names (on DockerHub), and their tags.

**Comment by mdrumond** [comment]
Rolled back the changes in in-memory analytics (it was a mistake)
About the new config, here it goes:
- DH_REPO="cloudsuite/graph-analytics" IMG_TAG="3.0" DF_PATH="./benchmarks/graph-analytics"
- DH_REPO="cloudsuite/twitter-graph-dataset" DF_PATH="./datasets/twitter-dataset-graph"

**Comment by neo-apz** [comment]
So `spark-base` images can be removed?
And just to be sure, this PR fixes #16 too, right?

**Comment by mdrumond** [comment]
Yes for both questions. Do you want me to do it and commit it?
